style.css

```diff
@@ -116,4 +118,836 @@
+    height: 48px;
+    background: #ff6b00;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 17px;
+    padding: 12px;
+    border: none;
+  }
+
+  /* Card 6 */
+
+  .card6 {
+    display: flex;
+    grid-row-start: 6;
+    grid-row-end: 7;
+    flex-direction: column;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+    background-image: url(./Images/Professional\ art\ printing.svg);
+    height: 500px;
+  }
+
+  .card6_title {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    margin-top: 40%;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card6_title6 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card6_paragraph {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    width: 85%;
+    color: #fff;
+    line-height: 24px;
+    margin-bottom: 20px;
+  }
+
+  .buttons6 {
+    display: flex;
+    width: 85%;
+    justify-content: flex-start;
+    list-style: none;
+    padding-left: 0;
+    gap: 10px;
+  }
+
+  .li_buttons6 {
+    background: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    padding: 10px 12px;
+    width: auto;
+    height: 40px;
+    border: none;
+    align-items: center;
+  }
+
+  .card6button {
+    margin-top: 20%;
+    width: 100%;
+    height: 48px;
+    background: #ff6b00;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 17px;
+    padding: 12px;
+    border: none;
+  }
+
+  /* Card 7 */
+
+  .card7 {
+    display: flex;
+    grid-row-start: 7;
+    grid-row-end: 8;
+    flex-direction: column;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+    background-image: url(./Images/Professional\ art\ printing.svg);
+    height: 500px;
+  }
+
+  .card7_title {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    margin-top: 40%;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card7_title7 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-size: 42px;
+    color: #fff;
+    font-weight: 400;
+    width: 85%;
+  }
+
+  .card7_paragraph {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    width: 85%;
+    color: #fff;
+    line-height: 24px;
+    margin-bottom: 20px;
+  }
+
+  .buttons7 {
+    display: flex;
+    width: 85%;
+    justify-content: flex-start;
+    list-style: none;
+    padding-left: 0;
+    gap: 10px;
+  }
```

Comment:
- [ ] **[OPTIONAL]**  Kindly we could avoid code repetition using: `.buttons1, .buttons2, ,buttons3, ....{ display: ....} `, and so on. 👍🏽